.github/workflows/node.js.yml

```diff
@@ -61,12 +62,12 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
-        env:
-          REACT_APP_BACKEND: ${{ secrets.BACKEND_URL }}
       - name: Install dependencies
         run: npm ci
       - name: Build the project
         run: npm run build --if-present
+        env:
+          REACT_APP_BACKEND: ${{ secrets.BACKEND_URL }}
```

Comment:
ahhh so the variable was defined in the wrong place?

Reply:
yup seems like it